tequilapi/endpoints/transactor.go

```diff
@@ -816,7 +817,7 @@ func AddRoutesForTransactor(
 			transGroup.POST("/stake/increase/async", te.SettleIntoStakeAsync)
 			transGroup.POST("/stake/decrease", te.DecreaseStake)
 			transGroup.POST("/settle/withdraw", te.Withdraw)
-			transGroup.GET("/token/:token/reward", te.TokenRewardAmount)
+			transGroup.GET("/token/:token/reward", a.TokenRewardAmount)
```

Comment:
this looks like it repeated, I'd say we should just keep `/affiliator` and bump the version, but maybe it's not such a good idea if we prefer to keep backwards compatibility

Reply:
Backward compatibility requires to leave this endpoint in "transactor" namespace